critiquebrainz/data/model/review.py

```diff
@@ -84,6 +85,11 @@ def text(self):
         """Returns text of the latest revision."""
         return self.last_revision.text  # latest revision
 
+    @property
+    def is_of_event(self):
+        """Returns whether the review's entity is an event"""
+        return self.entity_type == 'event'
```

Comment:
I don't think I like this property, way too specific. I'd rather keep things generic here and try not to refer to specific entity type, just compare to `entity_type` where it's used.